librz/analysis/meta.c

```diff
@@ -348,7 +345,7 @@ RZ_API void rz_meta_print(RzAnalysis *a, RzAnalysisMetaItem *d, ut64 start, ut64
 				if (!s) {
 					s = strdup(pstr);
 				}
-				if (rad) {
+				if (mode) {
```

Comment:
i'm not sure `if (mode)` has any particular meaning... This assumes that a specific enum value is 0 while others are not, but with enum I would not expect such behaviour. I'd prefer to see `if (mode == XXXXXX)`

Reply:
I suspect it means mode is not default one.

Reply:
yeah but... what is the *default* mode? Is there a default value of an enum?

I mean, I can see what this code does, but what I mean is that we should be explicit and make it more readable by using `mode == XXXX`, to ensure that even if at some point the order of the enum(or the values) change, this won't break.

Reply:
please fix this. In this specific case I think it should be `if (mode == RZ_OUTPUT_MODE_RIZIN`, but it may be different in other cases.